controller/src/main/java/io/pravega/controller/task/Stream/StreamTransactionMetadataTasks.java

```diff
@@ -57,6 +58,7 @@
  */
 @Slf4j
 public class StreamTransactionMetadataTasks implements AutoCloseable {
+    private static final int MAX_EXECUTION_TIME_MULTIPLIER = 1000;
```

Comment:
Please add the following comment here:

```
/*
 * We derive the maximum execution timeout from the lease time. We assume
 * a maximum number of renewals for the txn and compute the maximum execution
 * time by multiplying the lease timeout by the maximum number of renewals. This
 * multiplier is currently hardcoded because we do not expect applications to change
 * it. The maximum execution timeout is only a safety mechanism for application that
 * should rarely be triggered.
 */
```